### PR TITLE
Updated VsTestV2 task to version 10932935 And PTR Task to version 10925608

### PR DESCRIPTION
**Pull request by nikhilmsoft**
Updated TestAgent version from 10931517 to 10932935
https://mseng.visualstudio.com/AzureDevOps/_build/results?buildId=10932935&view=results

Updated PTR task version from 10864897to 10925608
https://mseng.visualstudio.com/AzureDevOps/_build/results?buildId=10925608&view=results

### CODE REVIEW

File: Tasks/VsTestV2/make.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
                 "dest": "./"
             },
             {
-                "url": "https://testexecution.blob.core.windows.net/testexecution/10931517/TestAgent.zip",
+                "url": "https://testexecution.blob.core.windows.net/testexecution/10925608/TestAgent.zip",
```
Comment (navin22) [inline Tasks/VsTestV2/make.json]: is this needed check ?

**Comment by navin22** [comment]
Bump up the PTR task version as well.

File: Tasks/VsTestV2/make.json (hunk 1 above)
Comment (nikhilmsoft) [reply Tasks/VsTestV2/make.json]: changes done

Comment (ShreyasRmsft) [reply Tasks/VsTestV2/make.json]: I see you have not merged your changes on mseng, on what basis did you update the version here?

Comment (navin22) [reply Tasks/VsTestV2/make.json]: Make sure the later build doesn't have changes in PTR Library which are needed for PTR task.
Approving It.

File: Tasks/VsTestV2/task.loc.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 161,
-    "Patch": 2
+    "Minor": 162,
```
Comment (ShreyasRmsft) [inline Tasks/VsTestV2/task.loc.json]: you seem to be down grading the version, it has to become 2.162.3

The task won't get updated nor will the package get published

File: Tasks/VsTestV2/task.json [hunk 3]
```diff
@@ -17,8 +17,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 161,
-        "Patch": 2
+        "Minor": 162,
```
Comment (ShreyasRmsft) [inline Tasks/VsTestV2/task.json]: you seem to be down grading the version, it has to become 2.162.3

The task won't get updated nor will the package get published

File: Tasks/VsTestV2/task.loc.json (hunk 2 above)
Comment (nikhilmsoft) [reply Tasks/VsTestV2/task.loc.json]: changed it to 2.162.1

File: Tasks/VsTestV2/task.json (hunk 3 above)
Comment (nikhilmsoft) [reply Tasks/VsTestV2/task.json]: changed it to 2.162.1

**Pull request merged**